app/javascript/src/components/Player.tsx

```diff
@@ -235,23 +237,46 @@ class Player extends React.Component<PlayerProps, PlayerState> {
     }
   }
 
+  private currValue(target: any, type: string) {
+    if (type === "stat") {
+      let currStats = clone(this.state.currentStats);
+      currStats = currStats.filter((k) => k.name === target);
+
+      if (currStats[0] === undefined) {
+        return 0
+      } else {
+        return currStats[0].value
+      }
+
+    } else if (type === "item") {
+      let currItems = clone(this.state.currentItems);
+
+      if (currItems.includes(target) === false) {
+        return 0
+      } else {
+        const countTarget = currItems.reduce((acc, val) => (val === target ? acc + 1 : acc), 0);
+        return countTarget
+      }
+    }
+  }
```

Comment:
Once I added the default value, it wouldn't allow me to edit the field for items. Would appreciate some eyes on this section.